tests/e2e/lib.sh

```diff
@@ -149,6 +149,7 @@ export_test_environment() {
     ci_export ROX_SCANNER_V4_SUPPORT "${ROX_SCANNER_V4_SUPPORT:-true}"
     ci_export ROX_CLOUD_CREDENTIALS "${ROX_CLOUD_CREDENTIALS:-true}"
     ci_export ROX_SCANNER_V4 "${ROX_SCANNER_V4:-false}"
+    ci_export ROX_CLOUD_SOURCES "${ROX_CLOUD_SOURCES:-true}"
```

Comment:
Need to also add this to the operator deployment in l.262 otherwise we may have issues on testes where operator deployment id used (e.g. openshift qa tests/ARO/ROSA)